### PR TITLE
Hw jdbc intro

### PR DESCRIPTION
**Pull request by ivanlavrinenko**

### CODE REVIEW

File: src/main/resources/init_db.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+);
+
+INSERT INTO Book (title, price) VALUES
+    ('The Great Gatsby', 29.99),
+    ('To Kill a Mockingbird', 19.95),
+    ('1984', 24.50),
+    ('Pride and Prejudice', 14.99),
+    ('The Catcher in the Rye', 17.75);
+
+SELECT * FROM book;
```
Comment (okuzan) [inline src/main/resources/init_db.sql:16]:
```suggestion
SELECT * FROM book;

```

File: src/main/java/mate/academy/dao/BookDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                Long id = resultSet.getLong("id");
+                String title = resultSet.getString("title");
+                BigDecimal price = resultSet.getObject("price", BigDecimal.class);
+                Book book = new Book();
+                book.setId(id);
+                book.setTitle(title);
+                book.setPrice(price);
+                books.add(book);
```
Comment (okuzan) [inline src/main/java/mate/academy/dao/BookDaoImpl.java:76]: image
[Checklist](https://github.com/mate-academy/jv-jdbc-intro/blob/main/checklist.md)

File: src/main/java/mate/academy/dao/BookDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+    @Override
+    public List<Book> findAll() {
+        String sql = "SELECT * FROM books";
+        List<Book> books = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(sql)) {
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                Long id = resultSet.getLong("id");
```
Comment (okuzan) [inline src/main/java/mate/academy/dao/BookDaoImpl.java:69]: image

**Review by okuzan — CHANGES_REQUESTED** [review]
Please go through the checklist before a review 😉

**Review by Ivan95kos — APPROVED** [review]
Great job

**Pull request closed without merging**